src/index.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```

Comment:
All that changed in this file was formatting. It's always a good idea to try to keep those types of changes to a minimum and only do them in special PRs with the sole purpose of reformatting. That way, whoever reviews your PR doesn't have to sift through formatting changes that have no semantic meaning to get to the significant parts.